distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Regex/Match_2.enso

```diff
@@ -21,18 +28,15 @@ type Match_2
     start : Integer -> Integer
     start self n = self.internal_regex_result.getStart n
```

Comment:
```suggestion
    start self n=0 = self.internal_regex_result.getStart n
```
If we default it to 0, I assume will then give the whole match start index.
Lets rename `n` to `group`

Reply:
Let's rename it to `utf16_start` as that would make it clearer. 

Can we also add a `start` and `end`, which give the graphene indexes.

```suggestion
    utf16_start self group=0 = self.internal_regex_result.getStart group

    start : Integer -> Integer
    start self group=0 = self.graphene_span group . start
```

Reply:
Done